src/FamilyHubs.ServiceDirectory.Core/FamilyHubs.ServiceDirectory.Core.csproj

```diff
@@ -16,8 +16,9 @@
 	</PropertyGroup>
 
 	<ItemGroup>
-	  <PackageReference Include="FamilyHubs.ServiceDirectory.Shared" Version="1.1.15" />
+	  <PackageReference Include="FamilyHubs.ServiceDirectory.Shared" Version="1.1.20" />
 	  <PackageReference Include="FamilyHubs.SharedKernel" Version="1.0.4" />
+	  <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```

Comment:
I think this is a legacy package (it hasn't been updated since 2018).

What functionality do you need from it?

I suspect there'll be a new way to achieve the same thing.

Reply:
resolved